internal/metrics/metricsManager.go

```diff
@@ -126,7 +127,7 @@ func NewMetricsManager(registry prometheus.Registerer) *MetricsManager {
 			Help:      counterDescriptionHostInstallationPhaseSeconds,
 			Buckets: []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10, 20, 30, 40, 50, 60, 90, 120, 150, 180, 210, 240, 270, 300, 360, 420, 480, 540,
 				600, 900, 1200, 1500, 1800, 2100, 2400, 2700, 3000, 3300, 3600},
-		}, []string{phaseLabel, resultLabel, openshiftVersionLabel, discoveryAgentVersionLabel}),
+		}, []string{phaseLabel, resultLabel, openshiftVersionLabel, discoveryAgentVersionLabel, hwVendorLabel}),
```

Comment:
If you're touching this code already, can we reduce the number of buckets to 10:
https://github.com/openshift/assisted-service/pull/494/files#diff-431d86d763ee7e2b80d9e2806444ac93R110
and
https://github.com/openshift/assisted-service/pull/494/files#diff-431d86d763ee7e2b80d9e2806444ac93R128
@avishayt

Reply:
@romfreiman How about:
1, 5, 10, 30, 60, 120, 300, 600, 900, 1200, 1800